### PR TITLE
Add docstring to sparql comments

### PR DESCRIPTION
**Pull request by tconway**

### CODE REVIEW

File: repository/models/queries/asset.py [hunk 1]
```diff
@@ -72,7 +90,14 @@
 }
 """
 
+# Gets asset by entity id
```
Comment (bnouvel-wio) [inline repository/models/queries/asset.py]: 'Gets' is maybe not the right word, we don't return the whole asset object but just identifiers to reference the asset.  Idem for next query.

File: repository/models/queries/set.py [hunk 2]
```diff
@@ -8,13 +8,18 @@
...
+# Used in conjunction with GENERIC_LIST query.
+# :param id_name: name of the variable for the id
+# :returns title: Title of set
 SET_LIST_EXTRA_IDS = """?title"""
 SET_LIST_EXTRA_QUERY = """
     OPTIONAL {{ ?{id_name} dcterm:title ?title . }}
 """
 
-# Template to create new sets.
+# Template to create new serts
```
Comment (bnouvel-wio) [inline repository/models/queries/set.py]: "serts" minor typo

**Comment by bnouvel-wio** [comment]
+1 - all good - I have added two minor comments.

**Comment by tconway** [comment]
Commit to address comments

**Pull request merged**